### PR TITLE
update dcat-usmetadata app version.

### PR DESCRIPTION
**Pull request by rshewitt**
# Pull Request

Related to [4274](https://github.com/GSA/data.gov/issues/4274)

## About

updated `requirements.ini` and `requirements.txt`

## PR TASKS

- [ ] The actual code changes.
- [ ] Tests written and passed.
- [ ] Any changes to docs?
- [x] Any new
[requirements versions](https://github.com/GSA/inventory-app/blob/main/requirements.in.txt)
to pull in?

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,21 +5,21 @@ Babel==2.10.3
...
 ckanext-datajson==0.1.20
-ckanext-dcat-usmetadata==0.4.2
+ckanext-dcat-usmetadata==0.5.0
 ckanext-envvars==0.0.3
 ckanext-googleanalyticsbasic==0.2.1
 ckanext-s3filestore @ git+https://github.com/keitaroinc/ckanext-s3filestore.git@caf88c0352ffe7b4432d3d55ddfb0a71249ceddd
 ckanext-saml2auth @ git+https://github.com/GSA/ckanext-saml2auth.git@c2b12a94430034c522b25d282323a064e2d6a03a
 ckanext-usmetadata==0.3.2
--e git+https://github.com/ckan/ckanext-xloader.git@c062f547e8e5dd80fd5dae6d184fe268ccb4b32e#egg=ckanext_xloader
+-e git+https://github.com/ckan/ckanext-xloader.git@16b84175005435d579607638dfe2056dda61af70#egg=ckanext_xloader
```
Comment (nickumia-reisys) [inline requirements.txt:22]: Hmmm...

**Review by nickumia-reisys — APPROVED** [review]
🥥

File: requirements.txt (hunk 1 above)
Comment (rshewitt) [reply requirements.txt:22]: is this something that shouldn't have changed?

Comment (nickumia-reisys) [reply requirements.txt:22]: I'm always concerned about changing requirements... but tests passed, so I don't have reason to complain too much 🤷

**Pull request merged**